.github/CONTRIBUTING.md

```diff
@@ -143,12 +146,13 @@ For further details about commit message guidelines please visit:
 
  * Note: any new commits to the `new_feature` branch on GitHub will automatically be included in the pull request, so make sure to only commit related changes to the same branch.
 
-## Pull Request Notes
-If you file a PR but you're still working on it, please add a [WIP] before the title text. This will tell the reviewers that you still intend to add more to the PR and we don't need to review it yet. When it's ready to be reviewed by a merger just edit the title text to remove the [WIP].
+## Drafts
+If you file a PR but you're still working on it, please make it a Draft. This will tell the reviewers that you still intend to add more to the PR and we don't need to review it yet. When it's ready to be reviewed for a merger, just click the `Ready for review` button. Please convert all your existing PRs with `[WIP]` in the title to Drafts. (If you do want reviews while it is a Draft, see "Comment requests" below.)
```

Comment:
Maybe link an image showing how to make a draft while making a pull request?

Maybe this and CR can also be mentioned in the PR template?

Reply:
Thank you. I have a plan to suggest more changes to the PR template, that will be in another PR.